types/tap.d.ts

```diff
@@ -0,0 +1,5 @@
+import { Tap } from './test'
+
+declare var tap: Tap;
+
+export = tap;
```

Comment:
I don't think that not having EOLs is in sync with the rest of node-tap source code?